src/hvm.c

```diff
@@ -1234,7 +1234,7 @@ Port expand(Net* net, Book* book, Port port) {
   Port old = vars_load(net, get_val(ROOT));
   Port got = peek(net, port);
   while (get_tag(got) == REF) {
-    boot_redex(net, new_pair(new_port(REF,get_val(got)), ROOT));
```

Comment:
Is this just to avoid one deref interaction every iteration of the expand?

Reply:
Since `get_tag(got) == REF`, we have `new_port(REF,get_val(got)) == got`.